### ISSUE TITLE
Support for wrapped ENS names

### ISSUE THREAD

**Issue by igor-sirotin** [status-im/status-desktop#13419]
# TLDR

Status doesn't support wrapped ENS names.

# User story

Recently I got an `sirotin.eth` ENS name with [ens.domains](https://ens.domains) app for my `0xf8a6c331a1140989d043fc066b47167e5132d9a3` wallet.

When I tried to add this ENS into Status app, I was rejected with "Username doesn't belong to you".

![image](https://github.com/status-im/status-desktop/assets/25482501/5bc2d9bf-87d4-4f07-82d9-ff49d7030d8e)

This happened because `ownerOf("sirotin.eth")` returned `0xd4416b13d2b3a9abae7acd5d6c2bbdbe25686401` (which I agree doesn't look like my wallet address).

This address was visible in the "Contract address" section in [ens.domains](https://ens.domains). Though my wallet address was set as "Owner". The app kindly added the callout "Some apps may show this address as owner". This is exactly what happened with Status. 

![image](https://github.com/status-im/status-desktop/assets/25482501/8cc63723-035a-47d4-8537-640c8d888682)

I ended up unwrapping my ENS name and then I was able to add it to Status. But I suppose we should support it out of the box.

**Comment by anastasiyaig** [comment]
@igor-sirotin ideally we dont want to have a ticket without team assignment, please pick one out of 3 :)
